### PR TITLE
Minimum spanning tree OOP C#

### PR DESCRIPTION
**Pull request by AnneEjsing**

### CODE REVIEW

File: benchmarks/spanning_tree/oop_c#/Program.cs [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string directory = System.IO.Directory.GetParent(Environment.CurrentDirectory).ToString();
+            int i = 0;
+            Edge[] edges = File.ReadAllLines($"../graph.csv")
+                                           .Select(v => Edge.FromCsv(v, i++))
+                                           .ToArray();
+            //int vertexCount = edges.Select(x => x.Node1).Concat(edges.Select(y => y.Node2)).Distinct().Count();
```
Comment (Caspersn2) [inline benchmarks/spanning_tree/oop_c#/Program.cs:18]:
```suggestion
```

File: benchmarks/spanning_tree/oop_c#/Program.cs [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+using System.IO;
+using System.Linq;
+
+namespace oop_c_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string directory = System.IO.Directory.GetParent(Environment.CurrentDirectory).ToString();
```
Comment (Caspersn2) [inline benchmarks/spanning_tree/oop_c#/Program.cs:13]: bruger du directory?

Comment (AnneEjsing) [reply benchmarks/spanning_tree/oop_c#/Program.cs:13]: Linjen skal bare slettes

Comment (AnneEjsing) [reply benchmarks/spanning_tree/oop_c#/Program.cs:13]:
```suggestion
```

File: benchmarks/spanning_tree/oop_c#/Program.cs [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+using System.Linq;
+
+namespace oop_c_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int i = 0;
+            Edge[] edges = File.ReadAllLines($"../graph.csv")
```
Comment (AnneEjsing) [inline benchmarks/spanning_tree/oop_c#/Program.cs:14]:
```suggestion
            Edge[] edges = File.ReadAllLines($"benchmarks/spanning_tree/graph.csv")

```

**Review by Nitram1995 — COMMENTED** [review]
Småting

**Review by Nitram1995 — APPROVED** [review]

**Review by Caspersn2 — APPROVED** [review]

**Pull request merged**